Directory.Build.props

```diff
@@ -30,8 +30,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.VisualStudio.Internal.MicroBuild.VisualStudio" Version="$(MicroBuildVersion)" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="5.0.3" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="3.11.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="6.0.0" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.0.0-6.final" PrivateAssets="all" />
```

Comment:
There is a '4.0.0' package which is published as unlisted. A 4.0.1 package should be published today or tomorrow depending on validation.

Reply:
The 4.0.1 update should be an easy follow-up once available 👍

Reply:
Sounds good

Reply:
4.0.1 packages are now available.

Reply:
➡️ Now using 4.0.1